packages/cli/src/commands/index.ts

```diff
@@ -123,6 +123,7 @@ export function registerCommands(program: CommanderStatic) {
   program
     .command('plugin:build')
     .description('Build a plugin')
+    .option('--minify', 'Minify the generated code')
```

Comment:
Could you sneak in an update to docs/local-dev/cli-commands.md while at it? Just listing this same option in the shell codeblock